gotopkg.go

```diff
@@ -1,204 +0,0 @@
-package main
-
-import (
-	"flag"
-	"fmt"
-	"github.com/groob/plist"
-	"io"
-	"io/ioutil"
-	"log"
-	"os"
-	"os/exec"
-	"strings"
-	"syscall"
-	"time"
-)
-
-const RECIPE_FAILED_CODE = 70
```

Comment:
Sure, most everything else in the code is camelCase, just this one case I think is "arguable" because it's the exact same variable name that Python uses, but I could also note that in a comment. And yep: know about the scoping of upper/lower start letter.